src/components/singular/JobGallery.js

```diff
@@ -144,9 +115,161 @@ const JobGallery = ({ type }) => {
           </div>
         </div>
       ))}
-      {isLoading && <SaveSpinner />}
     </div>
   );
 };
 
 export default JobGallery;
+
```

Comment:
How come this is all commented out? the endpoint has already been implemented. for testing?